diff --git a/node_modules/nostr-tools/.bun-tag-4096dbe7ab5f328e b/.bun-tag-4096dbe7ab5f328e
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/lib/cjs/abstract-pool.js b/lib/cjs/abstract-pool.js
index 5367aada9aac98c27c0f8b7effc98c5811fef20f..e489176a1fc519dbdf74b90894f2d125c17457be 100644
--- a/lib/cjs/abstract-pool.js
+++ b/lib/cjs/abstract-pool.js
@@ -297,7 +297,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/cjs/abstract-relay.js b/lib/cjs/abstract-relay.js
index 8485d42999cea74b0abdc8d969c711091f471381..cb137ffd44f6632a7a491aca7838ced9fb8798dc 100644
--- a/lib/cjs/abstract-relay.js
+++ b/lib/cjs/abstract-relay.js
@@ -296,7 +296,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/cjs/index.js b/lib/cjs/index.js
index 25ef3ca6eb5baa3b476ea8404e508454e354e71f..f8989e9e95ef01ca85261aadd7b3967b0a7001d2 100644
--- a/lib/cjs/index.js
+++ b/lib/cjs/index.js
@@ -822,7 +822,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/cjs/nip46.js b/lib/cjs/nip46.js
index d997d1b7b1c094df5640f92be8befbdcea4cfb1e..de5c30a012e3348d625c390424d5f3a650568863 100644
--- a/lib/cjs/nip46.js
+++ b/lib/cjs/nip46.js
@@ -501,7 +501,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/cjs/pool.js b/lib/cjs/pool.js
index 482cd7c8bcf24b52b25e76d166702c5e7311f092..28b4464bb3a66bca462e3ca4b7eaade0e0122251 100644
--- a/lib/cjs/pool.js
+++ b/lib/cjs/pool.js
@@ -382,7 +382,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/cjs/relay.js b/lib/cjs/relay.js
index 7c9ae89958ead78e391e36a5186abf0162306fec..3991699c1815474dd678e69c1dc3eb8e4873a175 100644
--- a/lib/cjs/relay.js
+++ b/lib/cjs/relay.js
@@ -384,7 +384,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/esm/abstract-pool.js b/lib/esm/abstract-pool.js
index b763baca6e9e3830a0300422b80c97a8726337ee..6ae832499cebd262f3cfba903f5179a0bf8b3568 100644
--- a/lib/esm/abstract-pool.js
+++ b/lib/esm/abstract-pool.js
@@ -271,7 +271,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/esm/abstract-relay.js b/lib/esm/abstract-relay.js
index d22c9a6d49e46c85f27ccdfbf0ac210caf31bd5a..575791c865710610662e4ce39d6b1b5402a1ee34 100644
--- a/lib/esm/abstract-relay.js
+++ b/lib/esm/abstract-relay.js
@@ -268,7 +268,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/esm/index.js b/lib/esm/index.js
index a54ded82f55e8511d16fa0535683cb7f3cf03026..05358a974889cb0dbccc142d7ff42a938527864a 100644
--- a/lib/esm/index.js
+++ b/lib/esm/index.js
@@ -762,7 +762,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/esm/nip46.js b/lib/esm/nip46.js
index e72bc58fea768ca40acdcb69e11c5d2e332af873..9e63a1ae09e6f549062a65c72b4e40ee0233b835 100644
--- a/lib/esm/nip46.js
+++ b/lib/esm/nip46.js
@@ -467,7 +467,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/esm/pool.js b/lib/esm/pool.js
index 364c39cb7cf412b2d49093c60f66d6236292b679..9648b4ec15b7e7f46292472ea16f08d057693a0f 100644
--- a/lib/esm/pool.js
+++ b/lib/esm/pool.js
@@ -354,7 +354,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/esm/relay.js b/lib/esm/relay.js
index 8ce41cb2d85d1dddda2097ffccd7cea056c0143e..66be51e3a1815ae47510a46001dff0b26568a7fd 100644
--- a/lib/esm/relay.js
+++ b/lib/esm/relay.js
@@ -354,7 +354,7 @@ var AbstractRelay = class {
         return reject(new Error(`no connection to ${this.url}, can't ping`));
       try {
         const sub = this.subscribe(
-          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+          [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
           {
             label: "<forced-ping>",
             oneose: () => {
diff --git a/lib/nostr.bundle.js b/lib/nostr.bundle.js
index a6f6202409e3b36520435b9b233438073ce542fe..839a381f691165c78d26788dba097fd49857f2a8 100644
--- a/lib/nostr.bundle.js
+++ b/lib/nostr.bundle.js
@@ -2977,7 +2977,7 @@ var NostrTools = (() => {
           return reject(new Error(`no connection to ${this.url}, can't ping`));
         try {
           const sub = this.subscribe(
-            [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 0 }],
+            [{ ids: ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"], limit: 1 }],
             {
               label: "<forced-ping>",
               oneose: () => {
